apps/prairielearn/src/pages/instructorInstanceAdminPublishing/components/ExtensionModifyModal.tsx

```diff
@@ -74,14 +75,7 @@ export function ExtensionModifyModal({
   };
 
   const validateEmails = async (value: string) => {
-    const uids = [
-      ...new Set(
-        value
-          .split(/[\n,\s]+/)
-          .map((uid) => uid.trim())
-          .filter((uid) => uid.length > 0),
-      ),
-    ];
+    const uids = parseUniqueValuesFromString(value);
```

Comment:
I don't quite understand why this isn't using `UniqueUidsFromStringSchema`?

Reply:
Discussed on call why this is.